ApplicationInsightsInternalLogger/src/main/java/com/microsoft/applicationinsights/internal/logger/InternalLogger.java

```diff
@@ -91,7 +91,7 @@ public int getValue() {
 
     private LoggerOutput loggerOutput = null;
 
-    private InternalLogger() {
+    InternalLogger() {
```

Comment:
does it need to be package accessible?

Reply:
private is redundant; enum constructors are always private. (another reason why I prefer to avoid the enum singleton pattern)

Reply:
👍 how about removing the constructor?